CyberEyes/CyberEyes/CustomizationPage.xaml.cs

```diff
@@ -16,5 +16,50 @@ void Handle_StartClicked(object sender, System.EventArgs e)
 		{
 			Navigation.PushAsync(new GamePage(), true);
 		}
+
+		void Handle_EasyTapped(object sender, System.EventArgs e)
+		{
+			easyFrame.BackgroundColor = Color.Accent;
+			var label = easyFrame.Content as Label;
+			label.Text = "● Easy";
```

Comment:
are we going to update and store the level of difficulty as an enum/var so that can be used when we generate the item list?